compiler/src/iree/compiler/Dialect/HAL/Transforms/ResolveTopologyQueries.cpp

```diff
@@ -0,0 +1,190 @@
+// Copyright 2024 The IREE Authors
```

Comment:
```suggestion
// Copyright 2025 The IREE Authors
```
(always use the current date for new files)